### PR TITLE
HPCC-25571 FR Translations for 8.X

### PR DESCRIPTION
**Pull request by kunalaswani**
Translations added for French (FR).

Signed-off-by: Kunal Aswani<kunal.aswani@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by kunalaswani** [comment]
@gfortil please review, thanks

File: esp/src/src/nls/fr/hpcc.ts [hunk 1]
```diff
@@ -125,8 +125,10 @@ export = {
     ContinueWorking: "Continuer à travailler",
     Copied: "Copié",
     Copy: "Copier",
+    CopyLogicalFiles: "Copier Les Fichiers Logiques Au Presse-papier ",
```
Comment (richardkchapman) [inline esp/src/src/nls/fr/hpcc.ts:128]: Capitalization does not match other phrases around here

**Comment by richardkchapman** [comment]
All seem to be capitalized on every word, which is not the case for the existing strings

File: esp/src/src/nls/fr/hpcc.ts (hunk 1 above)
Comment (kunalaswani) [reply esp/src/src/nls/fr/hpcc.ts:128]: I'll make the correction. good catch

Comment (kunalaswani) [reply esp/src/src/nls/fr/hpcc.ts]: I have double check with the strings for english and fixed the ones that do not need to be capitalized.

Comment (richardkchapman) [reply esp/src/src/nls/fr/hpcc.ts]: Still look to be a few capitalization inconsistencies...

**Comment by richardkchapman** [comment]
@kunalaswani @gfortil Capitalization is still all over the place

File: esp/src/src/nls/fr/hpcc.ts [hunk 2]
```diff
@@ -658,6 +670,7 @@ export = {
     Results: "Résultat(s)",
     Resume: "reprendre",
     RetainSuperfileStructure: "Conserver la structure du Superfichier",
+    RetainSuperfileStructureReason: "Retenir le structure de superfile doit être permis quand plus qu'un sous-fichier existe",
```
Comment (gfortil) [inline esp/src/src/nls/fr/hpcc.ts:673]: I meant to say superfichier not superfile

**Review by gfortil — CHANGES_REQUESTED** [review]

File: esp/src/src/nls/fr/hpcc.ts (hunk 2 above)
Comment (kunalaswani) [reply esp/src/src/nls/fr/hpcc.ts]: Change made.

**Review by richardkchapman — APPROVED** [review]

**Review by gfortil — APPROVED** [review]

**Pull request merged**